runtime/compiler/x/codegen/J9TreeEvaluator.hpp

```diff
@@ -72,7 +72,6 @@ class OMR_EXTENSIBLE TreeEvaluator: public J9::TreeEvaluator
    static TR::Register *integerLowestOneBit(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *integerNumberOfLeadingZeros(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *integerNumberOfTrailingZeros(TR::Node *node, TR::CodeGenerator *cg);
-   static TR::Register *integerBitCount(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
The [declaration of `longBitCount`](https://github.com/eclipse-openj9/openj9/blob/b096737a960e8b3601935f4bd756e2d3e2db96b9/runtime/compiler/x/codegen/J9TreeEvaluator.hpp#L79) that appears just below here should also be removed.